aftership.gemspec

```
@@ -15,8 +15,7 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
   s.requirements << 'none'
 
-  s.add_dependency 'faraday', '~> 1.0', '>= 1.0.1'
-  s.add_dependency 'faraday_middleware', '~> 1.0'
+  s.add_dependency 'faraday', '>= 2.0.0'
```

```suggestion
  s.add_dependency 'faraday', '>= 2.0.1'
```

as per https://github.com/AfterShip/aftership-sdk-ruby/pull/42#issuecomment-1333330131